spotify_dl/spotify_dl.py

```diff
@@ -40,14 +40,26 @@ def spotify_dl():
                         default=['bestaudio/best'])
     parser.add_argument('-m', '--skip_mp3', action='store_true',
                         help='Don\'t convert downloaded songs to mp3')
-    parser.add_argument("--url", action="store_true", help="Convert playlist link to uri first")
+    parser.add_argument('-l', '--url', action="store",
+                        help="Spotify Playlist link URL")
 
     args = parser.parse_args()
 
     if args.version:
         print("spotify_dl v{}".format(VERSION))
         exit(0)
 
+    if os.path.isfile(os.path.expanduser('~/.spotify_dl_settings')):
+        with open(os.path.expanduser('~/.spotify_dl_settings')) as file:
+            config = json.loads(file.read())
+
+        for key in config:
+            value = config[key]
```

Comment:
If we are iterating over all keys and getting values, then use the iterator (call `config.items`) and get both key and values at once